mlflow/java/client/src/test/java/org/mlflow/tracking/TestUtils.java

```diff
@@ -29,11 +29,10 @@ public static void assertMetric(List<Metric> metrics, String key, double value)
     Assert.assertTrue(metrics.stream().filter(e -> e.getKey().equals(key) && equals(e.getValue(), value)).findFirst().isPresent());
   }
 
-  public static void assertMetric(List<Metric> metrics, String key, double value, long runStart, long runEnd) {
```

Comment:
I could not find a place where this was being used. The signature conflicts with the proposed signature for `assertMetric` with timestamps and steps. Finally, this method seeks to enforce timestamp constraints that do not apply to MLflow metrics.